apps/tailwind-components/components/form/FieldInput.vue

```diff
@@ -21,6 +22,7 @@ defineProps<{
   label: string;
   required: boolean;
   data: columnValue;
+  options?: IInputValueLabel[];
```

Comment:
we would like this consistent between listbox, checkboxgroup, radiogroup, right?